modules/node_modules/@ncigdc/components/ClinicalCard.js

```diff
@@ -8,39 +8,84 @@ import Tabs from '@ncigdc/uikit/Tabs';
+const ClinicalCard = ({
+  p: {
+    case_id: caseId,
+    diagnoses = [],
+    family_histories: familyHistory = [],
+    demographic = {},
+    exposures = [],
+  },
+  activeTab,
+  setTab,
+  theme,
+}: {
+  p: Object,
+  activeTab: number,
+  setTab: Function,
+  theme: Object,
+}) => (
+  <Card
+    style={{ flex: 1 }}
+    title={
+      <Row style={{ justifyContent: 'space-between' }}>
+        <span>Clinical</span>
+        <DownloadButton
+          filename={`clinical.case-${caseId}`}
+          url={`${API}cases`}
+          activeText="Processing"
+          inactiveText="Export"
+          filters={{
+            op: 'and',
+            content: [
+              {
+                op: 'in',
+                content: {
+                  field: 'cases.case_id',
+                  value: [caseId],
+                },
+              },
+            ],
+          }}
+          fields={['case_id']}
+          dataExportExpands={['demographic', 'diagnoses', 'diagnoses.treatments', 'family_histories', 'exposures']}
+        />
+      </Row>
+    }
+  >
     <Tabs
       contentStyle={{ border: 'none' }}
       onTabClick={i => setTab(() => i)}
       tabs={[
         <p key="Demographic">Demographic</p>,
-        <p key="Diagnoses / Treatment">Diagnoses / Treatments ({(p.diagnoses || []).length})</p>,
-        <p key="Family Histories">Family Histories ({(p.family_histories || []).length})</p>,
-        <p key="Exposures">Exposures ({(p.exposures || []).length})</p>,
+        <p key="Diagnoses / Treatment">Diagnoses / Treatments ({diagnoses.length})</p>,
+        <p key="Family Histories">Family Histories ({familyHistory.length})</p>,
+        <p key="Exposures">Exposures ({exposures.length})</p>,
       ]}
       activeIndex={activeTab}
     >
       {activeTab === 0 &&
         <EntityPageVerticalTable
           thToTd={[
-            { th: 'ID', td: p.demographic.demographic_id },
-            { th: 'Ethnicity', td: p.demographic.ethnicity },
-            { th: 'Gender', td: p.demographic.gender },
-            { th: 'Race', td: p.demographic.race },
-            { th: 'Year of Birth', td: p.demographic.year_of_birth },
-            { th: 'Year of Death', td: p.demographic.year_of_death },
+            { th: 'ID', td: demographic.demographic_id },
+            { th: 'Ethnicity', td: demographic.ethnicity },
+            { th: 'Gender', td: demographic.gender },
+            { th: 'Race', td: demographic.race },
+            { th: 'Year of Birth', td: demographic.year_of_birth },
+            { th: 'Year of Death', td: demographic.year_of_death },
           ]}
           style={{ flex: 1 }}
         />
       }
       {activeTab === 1 &&
         <SideTabs
           contentStyle={{ border: 'none' }}
-          tabs={(p.diagnoses || []).map(x => <p key={x.diagnosis_id}>{x.diagnosis_id}</p>)}
-          tabContent={(p.diagnoses || []).map(x =>
+          tabs={diagnoses.map(x => <p key={x.diagnosis_id}>{x.diagnosis_id}</p>)}
```

Comment:
updated. Haven't moved it to be a container yet, but it handles the data structure.